crosswalks/BibTeX.csv

```diff
@@ -0,0 +1,74 @@
+Property,BibTeX (@softwareversion)
+codeRepository,repository
+programmingLanguage,
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,file
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version (1)
+storageRequirements,
+supportingData,
+author,author
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,date (2) (year) / (month)

```

Comment:
I have indicated the priority of fields with a number, but I can remove it if it is not consistent with other mappings.

Reply:
See above for the multiplicity of dates. For versions, honestly I don't know why schema.org defines both `version` and `softwareVersion`.

Other crosswalks just pick one (usually `softwareVersion`) or sometimes both, so you should do like they do.

We should have a policy for this eventually, but that's out of scope of this PR.

Reply:
For `softwareVersion`, I did like [CFF mapping](https://github.com/codemeta/codemeta/blob/3c30d4cb59864e9d89ee607edd1e5382ee582704/crosswalks/Citation%20File%20Format%201.2.0.csv#L19) which maps both, but I agree to just keep `softwareVersion`.